snap/frontend/config/index.js

```diff
@@ -6,3 +6,7 @@ export const explorerUrl = {
 	mainnet: 'https://symbol.fyi',
 	testnet: 'https://testnet.symbol.fyi'
 };
+
+export const Channels = {
+	confirmedAdded: 'confirmedAdded'
```

Comment:
Do you also plan to support cosigning transactions?

Reply:
yes, but no now. I will do the basic release first to support the transfer transaction only